.github/workflows/checks.yml

```
@@ -158,9 +158,11 @@ jobs:
         uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43
 
       - name: Gather version
+        # some cleanup of the sbt output to get the version sbt will use when publishing below
         run: |-
-          echo `git describe --tags | sed -e "s/v\(.*\)-\([0-9][0-9]*\).*/\\1-\\2-/"``git rev-parse HEAD | head -c8`-SNAPSHOT > ~/.version
-          cat ~/.version
+          sbt "akka-projection-core/version" --batch --no-colors | tail -n 1 | cut -f 2 -d ' ' | tr -d '\n' > ~/.version
```

Using `print` is a little simpler:

```suggestion
          sbt "print akka-projection-core/version" | tail -n 1 | tr -d '\n' > ~/.version
```